examples/named_entity_recognition/ner_wikigold_transformer.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>DO_LOWER_CASE should be True when using an uncased model. </p><p>Any reason we want to fix NUMBER_GPU? If it's not set, our utility function will automatically detect and use all available ones. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp/pull/449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Changed the DO_LOWER_CASE = True. </p><p> </p><p>It seems that the default value for NUMBER_GPU in different functions are inconsistent. For example, previous this could be set to "None" to use all GPUs. But recently the default value is "1" and it cannot be set to "None", otherwise there will be an error. I was using CPU to test the code and I have to set it as an integer even it's not necessary. I have changed the default values in NER python file so this will be hidden. </p>